Assets/XML/GameText/Global_CIV4GameText.xml

```diff
@@ -22110,10 +22119,10 @@
 	<TEXT>
 		<Tag>TXT_KEY_RELIGION_FOUNDED_FIRST</Tag>
 		<English>[ICON_BULLET]Founded by First to Discover [LINK=%s1]%s2[\LINK]</English>
-		<French>[ICON_BULLET]Religion fondée par le premier à découvrir : [COLOR_TECH_TEXT]%s1_TechName[COLOR_REVERT]</French>
-		<German>[ICON_BULLET]Durch den ersten Entdecker der Technologie [COLOR_TECH_TEXT]%s1_TechName[COLOR_REVERT] gegründet</German>
-		<Italian>[ICON_BULLET]Fondato dalla prima civiltà che scopre [NUM1:il:la:i:le] [COLOR_TECH_TEXT]%s1_TechName[COLOR_REVERT]</Italian>
-		<Spanish>[ICON_BULLET]Fundado por el primero en descubrir [NUM1:el:la:los:las] [COLOR_TECH_TEXT]%s1_TechName[COLOR_REVERT]</Spanish>
+		<French>[ICON_BULLET]Religion fondée par le premier à découvrir : [COLOR_TECH_TEXT]%s2[COLOR_REVERT]</French>
```

Comment:
This and the following 3 were changed on purpose. English was the only one having the text appearing as it has another tag, I've changed the wrong ones for Italian, French, German and Spanish.

Reply:
![BUG5DEU](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/a52a6242-666a-464d-b8ab-b190dd828aff)
![BUG5FRE](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/6d2da6c7-83a9-4caa-aaee-51346ff6ef6d)
![BUG5SPA](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/93a41b0a-2790-4e2f-86e5-9445b98ed70c)
![BUG5](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/95b957de-60d6-47c7-b981-0edf0fe92094)

Reply:
Another thing, I noticed in all languages we have 2 units with the same name: Camel Archer. One is a normal unit, the other a specific culture one. Normally it's not very good to have this situation, so I modified the second one for Italian into "Arab camel archer". Just to let you know, if you want to avoid the duplicate too. Ciao.

Reply:
![DoubleArcher](https://github.com/caveman2cosmos/Caveman2Cosmos/assets/141564082/efb6ac3c-b43b-4919-a54b-785c58f0f07c)

Reply:
You should add the [LINK=%s1]...[\LINK] around the %s2 where it is missing here while you're at it.
The link will add colour so the color part can be replaced by the link part.

Reply:
Done, you're right didn't notice that. Thanks for checking!